models/demos/ttnn_resnet/tt/ttnn_functional_resnet50_new_conv_api.py

```diff
@@ -575,7 +575,9 @@ def __init__(
             compute_kernel_config=compute_kernel_config,
         )  # num_classes = 1000
 
-        self.transpose_shards = True
+        self.transpose_shards = False
+        act_block_h_override = 1568
```

Comment:
these changes needed?

Reply:
Yes. I was getting an error that `act_block_h_override` was not defined. I think `is_wormhole_b0()` was False.